lang_tests/echo.bf

```diff
@@ -1,4 +1,5 @@
 [
+  ignore: JIT can't trace this yet
```

Comment:
Maybe clearer would be "yk's stopgap interpreter is currently unable to handle some of the necessary code"?

Reply:
Failures are in the trace compiler.

Reply:
"trace compiler is missing necessary functionality"?

Reply:
Works for me!